datasetinsights/stats/coco_stats.py

```diff
@@ -0,0 +1,237 @@
+import math
+
+import numpy as np
+import pandas as pd
+import pycocotools.coco
+from pycocotools.coco import COCO
+from tqdm import tqdm
+
+COCO_AREA_RANGES = [
+    [0 ** 2, 1e5 ** 2],  # all
+    [0 ** 2, 32 ** 2],  # small
+    [32 ** 2, 96 ** 2],  # medium
+    [96 ** 2, 1e5 ** 2],  # large
+    [96 ** 2, 128 ** 2],  # 96-128
+    [128 ** 2, 256 ** 2],  # 128-256
+    [256 ** 2, 512 ** 2],  # 256-512
+    [512 ** 2, 1e5 ** 2],
+]  # 512-inf
+
+COCO_AREA_DICT = {
+    0: "all",
+    1: "small",
+    2: "medium",
+    3: "large",
+    4: "96-128",
+    5: "128-256",
+    6: "256-512",
+    7: "512-inf",
+}
+
+
+def _get_empty_bbox_heatmap(coco_img_ids, coco_obj):
+    # calculate bbox shape according to largest image in the dataset
+    max_height, max_width = 0, 0
+    for idx in coco_img_ids:
+        img = coco_obj.loadImgs(idx)[0]
+        max_height = max(img["height"], max_height)
+        max_width = max(img["width"], max_width)
+
+    bbox_heatmap = np.zeros([max_height, max_width, 1])
+    return bbox_heatmap
+
+
+def _get_labeled_kpt_dict(coco_obj: pycocotools.coco.COCO):
+    kpt_dict = {}
+    coco_kpts = coco_obj.loadCats(coco_obj.getCatIds())[0]["keypoints"]
+    for key in coco_kpts:
+        kpt_dict[key] = 0.0
+    return kpt_dict
+
+
+def _get_bbox_area_dict():
+    bbox_dict = {}
+    for key in COCO_AREA_DICT:
+        bbox_dict[key] = 0
+    return bbox_dict
+
+
+def _get_bbox_relative_size(bbox_area, image_h, image_w):
+    bbox_relative_size = math.sqrt(bbox_area / (image_h * image_w))
+    return bbox_relative_size
+
+
+def get_labeled_keypoints(annotation_file):
```

Comment:
1. Why we are coupling annotation loading with getting the key points? can we move to "io" directory? 
2. Whats the purpose of this method? Refer this for docstring([PEP-257](https://www.python.org/dev/peps/pep-0257/)).